lib/browser/api/web-contents.js

```diff
@@ -556,6 +556,9 @@ WebContents.prototype._init = function () {
     if (prefs.webviewTag && prefs.contextIsolation) {
       electron.deprecate.log('Security Warning: A WebContents was just created with both webviewTag and contextIsolation enabled.  This combination is fundamentally less secure and effectively bypasses the protections of contextIsolation.  We strongly recommend you move away from webviews to OOPIF or BrowserView in order for your app to be more secure');
     }
+    if (prefs.nodeIntegration != null && !prefs.nodeIntegration) {
+      electron.deprecate.log('Deprecation Warning: Disabling nodeIntegration is deprecated. In Electron 12, all non-sandboxed renderers will have node integration enabled. See https://github.com/electron/electron/issues/23506 for more details.');
```

Comment:
Can we fit a "You should be using contextIsolation instead" kind of message in here as well?